packages/liveblocks-core/src/room.ts

```diff
@@ -1122,6 +1262,60 @@ function createCommentsApi<M extends BaseMetadata>(
     return body;
   }
 
+  async function getThreadsSince(options: { since: Date }) {
```

Comment:
What would be the use case of using this versus adding the `since` parameter on `getThreads`?

I could see getting resolved threads or threads having a certain metadata since a certain date as a possible call and this would make it confusing: should I use getThreads or getThreadsSince

Reply:
You can actually disregard my above comment, I understand the logic better now, it provides the changes in the return model!

I like it :)